apis.md

```diff
@@ -28,5 +28,8 @@ Assets API retrieves the backing cost data broken down by individual assets, e.g
 
 __[/savings](https://docs.google.com/document/d/1h_LQuTdwzbzcSonZ49w0lktaaHsulNsIj-O0OZR12fc/edit)__
 
-Exposes a set of APIs for cost optimization insights, e.g. pod-rightsizing and more. 
+Exposes a set of APIs for cost optimization insights.
 
+__[/savings/requestSizing](./api-request-right-sizing.md)
```

Comment:
do you like this in its own separate doc vs having all in one Savings API page?

potentially, related... feels like its an opportunity to move that Google doc to this repo...

Reply:
Moved, let me know what you think.